development/horizontal_loader/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Horizontal Loader</title>
+
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body>
+    <script src="index.js"></script>
+    <script>
+      // These setTimeout functions are needed as CSS animaitions do not apply
+      // straight away to dynamically created elements.
+      HorizontalLoader.init( { showIntermediateBlocks: true } );
+      setTimeout( function() {
+        HorizontalLoader.show().set( 60, 3 );
+
+        setTimeout( function() {
+          HorizontalLoader.set( 100, 2 );
+
+          setTimeout( function() {
+            HorizontalLoader.hide();
+          }, 2000 );
```

Comment:
The issue requires you to make the loader vanish after 350ms. Change the timeout from 2000 to 2350.

Reply:
Thanks @utkarsh23, I will change this value. I had misread the specs as the horizontal loader should take 350ms to vanish, not that it should vanish 350ms after reaching 100%